api/src/controllers/ministry.ts

```diff
@@ -19,28 +19,17 @@
 'use strict';
 
 import { errorWithCode, logger } from '@bcgov/common-nodejs-utils';
-import { Response } from 'express';
+import { Request, Response } from 'express';
+import DataManager from '../db';
+import shared from '../libs/shared';
 
-// const dm = new DataManager(shared.pgPool);
+const dm = new DataManager(shared.pgPool);
 
-export const fetchMinistrySponsors = async (
-  { headers, params }: { headers: any, params: any }, res: Response
-): Promise<void> => {
-  // const { NamespaceModel } = dm;
-  // const { profileId } = params;
-  // console.log(headers);
+export const fetchMinistrySponsors = async (req: Request, res: Response): Promise<void> => {
+  const { MinistryModel } = dm;
   try {
-    // const results = await NamespaceModel.findForProfile(Number(profileId));
-    const results = [
-      {
-        name: 'Citizens\s Services',
-        code: 'CITZ',
-      },
-      {
-        name: 'Transportation & Infrastructure',
-        code: 'TRAN',
-      },
-    ]
+    const results = await MinistryModel.findAllMinistries();
```

Comment:
You could go with the base class' `findAll()` also not not implement `findAllMinistries()`.

Reply:
To use the base findAll() would require updating the DB schema as [get_bus_org](https://github.com/bcgov/platform-services-registry/blob/master/db/sql/0001.sql#L32-L37) currently doesn't populate an archived field.